infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/provision/Openshift4TrustedCAProvisioner.java

```diff
@@ -43,17 +47,25 @@
 
   private final String certificateMountPath;
   private final boolean trustedStoreInitialized;
+  private final String caBundleConfigMap;
   private final String configMapName;
   private final Map<String, String> configMapLabelKeyValue;
+  private final CheInstallationLocation cheInstallationLocation;
+  private final OpenShiftClientFactory clientFactory;
 
   @Inject
   public Openshift4TrustedCAProvisioner(
+      CheInstallationLocation cheInstallationLocation,
```

Comment:
Usualy we put class type parameters after the `@Named` strings.

Reply:
Fixed in commit https://github.com/eclipse/che/pull/18264/commits/2d143ee3ed3e9643917e3844503597ada90bf41d